package.json

```
@@ -8,17 +8,17 @@
   },
   "bugs": "https://github.com/asyncapi/cli/issues",
   "dependencies": {
-    "@asyncapi/avro-schema-parser": "^3.0.22",
-    "@asyncapi/bundler": "^0.5.2",
-    "@asyncapi/converter": "^1.4.19",
+    "@asyncapi/avro-schema-parser": "^3.0.23",
+    "@asyncapi/bundler": "^0.5.3",
+    "@asyncapi/converter": "^1.4.20",
     "@asyncapi/diff": "^0.4.1",
     "@asyncapi/generator": "^1.17.25",
     "@asyncapi/modelina": "^3.5.1",
-    "@asyncapi/openapi-schema-parser": "^3.0.22",
+    "@asyncapi/openapi-schema-parser": "^3.0.23",
     "@asyncapi/optimizer": "^1.0.2",
-    "@asyncapi/parser": "^3.0.14",
-    "@asyncapi/protobuf-schema-parser": "^3.2.12",
-    "@asyncapi/raml-dt-schema-parser": "^4.0.22",
+    "@asyncapi/parser": "^3.0.16",
```

I guess it should be enough with reverting this change in favour of https://github.com/asyncapi/cli/pull/1456 and also updating the title of the PR, right?
```suggestion
    "@asyncapi/parser": "^3.0.14",
```